WooCommerce/Classes/ViewRelated/Blaze/BlazeEditAd/BlazeEditAdView.swift

```diff
@@ -0,0 +1,289 @@
+        static let title = NSLocalizedString(
+            "blazeEditAdView.title",
+            value: "Edit Ad",
+            comment: "Title for the Blaze Edit Ad screen."
+        )
+        static let cancel = NSLocalizedString(
+            "blazeEditAdView.cancel",
+            value: "Cancel",
+            comment: "Cancel button in the Blaze Edit Ad screen."
+        )
+        static let save = NSLocalizedString(
+            "blazeEditAdView.save",
+            value: "Save",
+            comment: "Save button in the Blaze Edit Ad screen."
+        )
+        enum Image {
+            static let changeImage = NSLocalizedString(
+                "blazeEditAdView.image.changeImage",
+                value: "Change image",
+                comment: "Change image button title in the Blaze Edit Ad screen."
+            )
+        }
+        enum Tagline {
+            static let title = NSLocalizedString(
+                "blazeEditAdView.tagline.title",
+                value: "Tagline",
+                comment: "Tagline title text in the Blaze Edit Ad screen."
+            )
+            static let placeholder = NSLocalizedString(
+                "blazeEditAdView.tagline.placeholder",
+                value: "Title for the Blaze Ad",
+                comment: "Placeholder for Tagline text field in the Blaze Edit Ad screen."
+            )
+        }
+        enum Description {
+            static let title = NSLocalizedString(
+                "blazeEditAdView.description.title",
+                value: "Description",
+                comment: "Description title text in the Blaze Edit Ad screen."
+            )
+            static let placeholder = NSLocalizedString(
+                "blazeEditAdView.description.placeholder",
+                value: "Description text for the Blaze Ad",
+                comment: "Placeholder for Description text field in the Blaze Edit Ad screen."
+            )
+        }
+        static let regenerateByAI = NSLocalizedString(
+            "blazeEditAdView.regenerateByAI",
+            value: "Regenerate by AI",
+            comment: "Regenerate by AI button title in the Blaze Edit Ad screen."
+        )
+    }
+}
+
+private enum Layout {
+    static let imageBlockInsets: EdgeInsets = .init(top: 24, leading: 16, bottom: 0, trailing: 16)
+    static let insets: EdgeInsets = .init(top: 24, leading: 16, bottom: 16, trailing: 16)
+
+    static let parentVerticalSpacing: CGFloat = 24
+    static let childVerticalSpacing: CGFloat = 16
+    static let textBlockVerticalSpacing: CGFloat = 8
+
+    static let cornerRadius: CGFloat = 8
+    static let strokeWidth: CGFloat = 1
+    static let textFieldContentInsets: EdgeInsets = .init(top: 10, leading: 10, bottom: 10, trailing: 10)
+    static let placeholderInsets: EdgeInsets = .init(top: 18, leading: 16, bottom: 18, trailing: 16)
+
+    enum Image {
+        static let imageViewSize: CGFloat = 148
+        static let cornerRadius: CGFloat = 7.4
```

Comment:
Nit: although this reflects the design perfectly, I wonder if it was a design mistake because the number is a bit odd. How about changing it to 8 as it's close to the design, consistent with other parts of the app, and follows the [8-point grid rule](https://spec.fm/specifics/8-pt-grid) for a pixel-perfect UI?

Reply:
I agree. Thanks for the link. Done in 5bd3e63d519de2318bea992a92d6b325b0eb9d15